### DISCUSSION TITLE
Εγκατάσταση Arch Linux σε dual boot ή usb drive

### DISCUSSION THREAD

**Discussion by stsivoula** [courses-ionio/hci#1633] [Announcements]
Καλησπέρα σας,

Εξαιτίας μεγάλου φόρτου εργασιών δεν μπορώ να παρακολουθώ τακτικά το μάθημα και επομένως δεν μπορώ να γνωρίζω εάν έχει απαντηθεί η παρακάτω ερώτηση. Θα ήθελα να ξέρω εάν στο πλαίσιο του μαθήματος μπορώ να κάνω την εγκατάσταση των Arch linux στο σύστημά μου με dual boot. Έχω ήδη διαβάσει τις οδηγίες και δεν είδα πουθενά εάν έχετε κάποιο πρόβλημα με την ύπαρξη δύο λειτουργικών συστημάτων στο ίδιο σύστημα. Απλά θα ήθελα να ξεκαθαρίσω πως μιλάω για dual boot και όχι για virtual machine και θα ήθελα να γνωρίζω εάν θεωρείτε το dual boot τον βέλτιστο τρόπο.

**Comment by Vasilakious** [comment]
Ναι μπορείς και με dual-boot να τα εγκαταστήσεις τα Arch Linux. Αν θυμάμαι σωστά δεν ήθελε ο κύριος Virtual Machine.

**Reply by stsivoula** [reply]
Αυτό το γνωρίζω, απλά έμαθα πως ο κύριος προτείνει το live usb από το dual boot και ήθελα να μάθω αν θα έχω βαθμολογικές κυρώσεις με το dual boot

**Reply by Vasilakious** [reply]
Δεν αναφέρθηκε ποτέ σε κάτι τέτοιο όσον αφορά τα Dual-Boot.

**Reply by stsivoula** [reply]
Από όσο γνωρίζω το έχει πει προφορικά σε μάθημα πως καλύτερα να γίνει η εγκατάσταση σε live usb

**Reply by Vasilakious** [reply]
Βαθμό δεν κόβει για Dual-Boot don't worry.

**Reply by epidrome** [reply]
σωστά, η σύσταση για εξωτερικό αποθηκευτικό μέσο έγινε γιατί έχει μικρότερο ρίσκο να χαλάσετε το βασικό σας μηχάνημα

**Comment by KaterinaLaz** [comment]
Με βάση την απάντηση του [εδώ](https://github.com/courses-ionio/hci/discussions/1616#discussioncomment-1565440). ΤΟ θέμα είναι να μην ειναι VM  
Βέβαια αυτό δεν είναι ξεκάθαρο...

**Reply by runtheorun-exe** [reply]
Εν τέλει σε τι απάντησε; Είναι απάντηση αν τελειώνει με ερωτηματικό; Έχει πει ένα solid, crystal clear, plain and simple "Ναι, εφόσον δεν είναι VM είναι δεκτό, ακόμα και αν γίνει Dual Boot/Live USB" ή "Όχι, πρέπει να κάνεις τα Arch το daily driver σου, να ξεχάσεις τα Windows/Mac για να περάσεις το μάθημα με 5"; Μπορεί να τα έχω παίξει και να μην καταλαβαίνω ακριβώς τι λέει στην απάντηση, αλλά δεν απαντήθηκε καμία ερώτηση με σιγουριά, κατ εμέ.

**Comment by AlexandrosV20** [comment]
εγω που προσπαθησα να το κάνω σε δισκο στην αρχη με παλια windows μεσα δεν το επαιρνε λογω του οτι ο δισκος πρεπει να είναι τελείως  unallocated γιατι αλλιως δεν θα στο παρει το ινσταλ σωστα... μετα το περασα σε στικακι... απο οσα εχει πει μπορεις να το βαλεις και σε vm απλα δεν θα βαθμολογηθει αριστα

**Reply by stsivoula** [reply]
Δεν χρειάζεται να είναι όλος ο δίσκος unallocated. Μπορείς να κάνεις νέο partition για να περάσεις το linux. Είναι εύκολη διαδικασία και καθόλου χρονοβόρα. Απλά μετά πρέπει να προσέξεις να κάνεις την εγκατάσταση στο σωστό unallocated partition για να μην διαγράψεις καταλάθος το main os σου

**Reply by AlexandrosV20** [reply]
εγω το πηγαινα για ινσταλ σε ολοκληρο τον δισκο και οχι σε παρτισιον του για υτο και σου ειπα αυτο... νομιζα οτι θα το παρει οπως κανεις φορματ στα windows... ασε που καθε φορα που μπουταρω τα windows μετα πειραζει τα settings ten bios και πρεπει να τα ξαναγυρισεις σε αυτα που θελουν τα arch
